ocaml-lsp-server/src/compl.ml

```diff
@@ -259,13 +264,26 @@ let complete (state : State.t)
       match Document.kind doc with
       | `Other -> Fiber.return None
       | `Merlin merlin ->
+        let resolve =
+          let capabilities = State.client_capabilities state in
+          match
+            let open Option.O in
+            let* td = capabilities.textDocument in
+            let* compl = td.completion in
+            let* item = compl.completionItem in
+            item.resolveSupport
+          with
+          | None -> false
+          | Some { properties } ->
+            List.mem properties ~equal:String.equal "documentation"
+        in
```

Comment:
would it make sense to have a module-wide state - a lazy value - that would compute this value once instead of doing it on every completion

Reply:
Sure, it would make sense. We have a `Diganostics.t` that stores the results of computing capabilities. It will make sense here too.